src/components/Ticket/utils/timeUtils.ts

```diff
@@ -0,0 +1,24 @@
+export const formatDate = (date: Date, short: boolean = false) => {
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+
+    const amOrPm = hours >= 12 ? 'PM' : 'AM';
+    const formattedHours =
+        hours % 12 === 0 ? 12 : hours < 12 ? `0${hours}` : hours % 12;
+    const formattedMinutes = minutes < 10 ? `0${minutes}` : minutes;
+
+    const dateOptions: Intl.DateTimeFormatOptions = {
```

Comment:
can we extend some more date types

Reply:
Resolved in 572f8e063a96626c489e7124ef0a2f7cab1534b9